tensorflow_addons/layers/normalizations.py

```diff
@@ -16,8 +16,11 @@
 # =============================================================================
 
 import logging
+from typeguard import typechecked
```

Comment:
please put it behind `import tensorflow as tf`

Reply:
or move it into `types.py`

Reply:
Sure thing